### ISSUE TITLE
Vulnerabilities in the Docker image mcr.microsoft.com/dotnet/aspnet:6.0

### ISSUE THREAD

**Issue by sebastianbk** [dotnet/dotnet-docker#3998]
Hello,

We are using the Microsoft-provided Docker image called `mcr.microsoft.com/dotnet/aspnet:6.0` in production. This is the current LTS image for the ASP.NET Core Runtime.

Unfortunately, this image contains the following two vulnerabilities:

- CVE-2022-2509 - gnutls28
- CVE-2021-46828 - libtirpc, libtirpc

The vulnerabilities were found by the container scanning tool in Amazon Web Services. Please see the details of the findings below.

Would you be able to provide an update to this Docker image?

We rely on this image in production and cannot move ahead with our SOC 2 certification because of this issue.

Please let me know if I can be of any assistance in resolving this issue.

<img width="495" alt="image" src="https://user-images.githubusercontent.com/660576/184820890-267dc240-0fe7-4dc3-bb14-d151ce043996.png">

<img width="495" alt="image" src="https://user-images.githubusercontent.com/660576/184820932-4ceb0d9b-860e-4c04-b8cf-1ec8438b973f.png">

**Comment by vsarunov** [comment]
To add to the information here are the vulnerabilities in debian security tracker:

https://security-tracker.debian.org/tracker/CVE-2021-46828
https://lists.debian.org/debian-security-announce/2022/msg00170.html

**Comment by mthalman** [comment]
We're aware of these vulnerabilities. They come from the base Debian container image. See commands below:

```console
> docker run --rm debian:bullseye-slim apt list libgnutls30

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Listing...
libgnutls30/now 3.7.1-5+deb11u1 amd64 [installed,local]
```

> Version 3.7.1-5+deb11u1 is vulnerable. Fixed in version 3.7.1-5+deb11u2.

```console
> docker run --rm debian:bullseye-slim apt list libtirpc3

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Listing...
libtirpc3/now 1.3.1-1 amd64 [installed,local]
```

> Version 1.3.1-1 is vulnerable. Fixed in version 1.3.1-1+deb11u1.

We rely on the Linux distro maintainers to update their base images when relevant and necessary. As soon as new base images from Debian are available, the .NET images will automatically be rebuilt.

**Comment by mthalman** [comment]
The .NET images have been rebuilt in response to new Debian base images being available. They now contain the fixed versions of these packages.
